fastparquet/writer.py

```diff
@@ -223,7 +226,7 @@ def infer_object_encoding(data):
         return 'json'
     elif all(isinstance(i, bool) for i in head):
         return 'bool'
-    elif all(isinstance(i, int) for i in head):
+    elif all(isinstance(i, integer_types) for i in head):
```

Comment:
Good catch. This includes `long`, correct?

Reply:
Yes, `integer_types` is `(int, long)` in Python2, and `int` in Python3